style/properties/longhands/inherited_box.mako.rs

```diff
@@ -12,6 +12,7 @@ ${helpers.single_keyword(
     gecko_ffi_name="mVisible",
     spec="https://drafts.csswg.org/css-box/#propdef-visibility",
     gecko_enum_prefix="StyleVisibility",
+    servo_restyle_damage="repaint",
```

Comment:
`visibility` can establish an independent formatting context

Reply:
Actually, no, it can't. I think I saw the `visible` and `hidden` values and I thought about `overflow`.

But anyways, `visibility: collapse` can affect layout.